Terminal.Gui/Text/Autocomplete/PopupAutocomplete.cs

```diff
@@ -187,7 +187,7 @@ public override bool ProcessKey (Key a)
             return ReopenSuggestions ();
         }
 
-        if (closed || (Suggestions.Count == 0))
+        if (closed || Suggestions.Count == 0)
```

Comment:
Opportunity for use of multi-condition switch statement. (`switch ( closed, Suggestions )`)